cobc/parser.y

```diff
@@ -407,6 +407,46 @@ emit_statement (cb_tree x)
 	}
 }
 
+#if 0
+/* Uncomment one of these functions if needed. */
+
+/* Insert a statement before the current statement. */
+static COB_INLINE COB_A_INLINE void
+emit_statement_before (cb_tree x)
+{
+	if (!skip_statements){
+		emit_statement(x);
+		struct cb_list * l1 = CB_LIST (current_program->exec_list);
+		struct cb_list * l2 = CB_LIST (l1->chain);
+		if (l2){
+			l1->chain = l2->chain;
+			l2->chain = CB_TREE(l1);
+			current_program->exec_list = CB_TREE (l2);
+		}
+	}
+}
+
+/* Drop the current statement. */
+static COB_INLINE COB_A_INLINE void
+drop_last_statement (void)
+{
+	if (!skip_statements){
+		if (current_program->exec_list){
+			struct cb_list * l1 = CB_LIST (current_program->exec_list);
+			current_program->exec_list = l1->chain;
+		}
+	}
+}
+
+/* Insert a statement to replace the current statement. */
+static COB_INLINE COB_A_INLINE void
+replace_last_statement (cb_tree x)
+{
+	drop_last_statement ();
+	emit_statement (x);
+}
+#endif
+
```

Comment:
we should add it if/when they are actually used, not in a previous commit

Reply:
Is there some kind of place where we can keep such functions for later when we need them ? If I remove them now, they will have to be rewritten from scratch when needed...

Reply:
Sure, you can add as much feature branches in git (locally or also in this mirror) as you want :-)

Reply:
Removed